embargo-list.md

```diff
@@ -0,0 +1,121 @@
+  issues up to the present day.
+* Be an **active participant and contributor** in the Erlang Ecosystem
+  community.
+
+### 3.2 Qualifying roles (meet at least one)
+
+| Category                  | Examples                                                          |
+| ------------------------- | ----------------------------------------------------------------- |
+| **Distributors**          | Linux/BSD distros, Docker/OCI image publishers, Nixpkgs, Homebrew |
+| **Framework maintainers** | Phoenix, Nerves                                                   |
+| **End‑user projects**     | RabbitMQ, Riak, VerneMQ, MongooseIM                               |
+| **Commercial vendors**    | SaaS/PaaS providers, appliance vendors, embedded product makers   |
+{: .table.table-striped.table-borderless }
+
+> Membership is **not** a right; acceptance is at the sole discretion of the
+> **EEF CNA**.
+>
+> **Removal:** If a participant later fails to meet any baseline requirement,
+> they will be unsubscribed from the List.
+
+## 4. Enrollment Process
+
+1. Send an email to **[cna@erlef.org](mailto:cna@erlef.org)** containing:
+
+   * GitHub **user** ID to be added to the List.
+   * A statement that you have read and will follow this policy (§6–§10).
+   * (Optional) A public PGP key for encrypted one‑off communications.
+2. Applications are reviewed on a **continuous basis**; additional information
+   may be requested.
+3. Approved participants receive an invitation to the private GitHub repository
+   `cna‑embargo`.
+
+## 5. List Operations
+
+* All **CVE Records** reside in the private repository. Configure your GitHub
+  **Watch** settings to receive notifications.
+* Only EEF CNA personnel can modify repository contents. Participants may
+  coordinate in **GitHub Discussions** linked to each record.
+* Do **not** mirror, fork, or redistribute the repository.
+
+## 6. Confidentiality Requirements
+
+Participants **must**:
+
+1. Keep all information confidential until the CVE Record is **published**.
+2. Limit internal sharing strictly to staff who *need‑to‑know* for patching or
+   mitigation.
+3. Refrain from hinting publicly (e.g., vague tweets, CVE reservations, release
+   notes) that a security fix is forthcoming.
+4. **Do not** create public issues, pull requests, or commits that reveal any
+   part of the vulnerability—even if the commit message omits details.
+
+## 7. Embargo Timelines
+
+| Scenario                                         | Heads‑up window                                                     |
+| ------------------------------------------------ | ------------------------------------------------------------------- |
+| **Information already public**                   | Immediate publication.                                              |
+| **Erlang / Elixir / Gleam core vulnerabilities** | Up to **7 days**.                                                   |
```

Comment:
:warning: This would mean that those projects agree that we give one weeks heads up before publication.